src/main/java/duke/command/InvalidCommand.java

```diff
@@ -0,0 +1,32 @@
+package duke.command;
+
+import duke.util.Storage;
+import duke.util.TaskList;
+import duke.util.Ui;
+
+/**
+ * Represents an invalid command.
+ */
+public class InvalidCommand extends Command {
```

Comment:
I like how you have a separate class for an invalid command; I would not have thought to handle invalid commands like this.